libs/execution_context/ebpf_core.c

```diff
@@ -2124,6 +2176,11 @@ _ebpf_core_map_find_element(ebpf_map_t* map, const uint8_t* key)
 {
     ebpf_result_t retval;
     uint8_t* value;
+    // Workaround for bug in ebpf-verifier that permits NULL map. Remove when fixed.
```

Comment:
Add issue number to comment so one can track when it's fixed.
Same in line 2195 etc.

Reply:
Fixed.